Project1/Index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="styles.css">
+  <title>Document</title>
+</head>
+
+<body>
+
+  <figure>
+    <img src="images/octopus.jpg" alt="A picture of an octopus">
+    <figcaption>A picture of an octopus</figcaption>
+  </figure>
+  <h1>Yes, the Octopus Is Smart as Heck. But Why?
```

Comment:
You need to close this `h1` tag:

```suggestion
  <h1>Yes, the Octopus Is Smart as Heck. But Why?</h1>
```